recipes/next/README.md

```diff
@@ -30,10 +31,12 @@ You can do this for any route on the application, **even if the page doesn't exi
 
 To adopt this recipe you will need to:
 
-- **IMPORTANT** Add authentication to `/edit` routes. This can be done by modifying the example API routes in `/app/api/puck/route.ts` and server component in `/app/[...puckPath]/page.tsx`. **If you don't do this, Puck will be completely public.**
+- **IMPORTANT** Add authentication to `/edit` routes. This can be done by modifying the example API routes in `/app/api/puck/route.ts` and server component in `/app/edit/[...puckPath]/page.tsx`. **If you don't do this, Puck will be completely public.**
```

Comment:
```suggestion
- **IMPORTANT** Add authentication to `/edit` routes. This can be done by modifying the example API routes in `/app/puck/api/route.ts` and server component in `/app/puck/[...puckPath]/page.tsx`. **If you don't do this, Puck will be completely public.**
```

Reply:
Fixed